packages/http/libraries/core/src/decorators/Delete.ts

```diff
@@ -0,0 +1,6 @@
+import { RequestMethodType } from '../models/RequestMethodType';
+import { requestMethod } from './RequestMethod';
+
+export function delet(path?: string): MethodDecorator {
```

Comment:
```suggestion
export function delete(path?: string): MethodDecorator {
```

Reply:
We have a problem here: `delete` is a reserved word in ts

Reply:
Fixed